contracts/0.7.x/MultipleArbitrableTokenTransactionWithAppeals.sol

```diff
@@ -177,12 +176,12 @@ contract MultipleArbitrableTokenTransactionWithAppeals is IArbitrable, IEvidence
     modifier onlyValidTransactionCD(uint256 _transactionID, Transaction calldata _transaction) {
         require(
             transactionHashes[_transactionID - 1] == hashTransactionStateCD(_transaction), 
-            "Transaction doesn't match stored hash"
+            "Transaction doesn't match stored hash."
             );
         _;
     }
 
-    /** @dev Create a transaction. UNTRUSTED.
+    /** @notice Create a transaction. UNTRUSTED.
```

Comment:
We use `@dev` because our messages are intended for developer tooling, not UIs.

Reply:
Ok, just changed it.